docker-compose.yml

```diff
@@ -285,6 +285,34 @@ services:
       - _APP_DB_PASS
       - _APP_LOGGING_PROVIDER
       - _APP_LOGGING_CONFIG
+  
+  appwrite-worker-builds:
+    entrypoint: worker-builds
+    container_name: appwrite-worker-builds
+    build:
+      context: .
+    networks:
+      - appwrite
+    volumes: 
+      - ./app:/usr/src/code/app
+      - ./src:/usr/src/code/src
+    depends_on:
+      - redis
+      - mariadb
+    environment:
+      - _APP_ENV
+      - _APP_REDIS_HOST
+      - _APP_REDIS_PORT
+      - _APP_REDIS_USER
+      - _APP_REDIS_PASS
+      - _APP_DB_HOST
+      - _APP_DB_PORT
+      - _APP_DB_SCHEMA
+      - _APP_DB_USER
+      - _APP_DB_PASS
+      - _APP_LOGGING_PROVIDER
+      - _APP_LOGGING_CONFIG
+      - _APP_EXECUTOR_SECRET
```

Comment:
This also needs to be added to the docker-compose template for installation.

Reply:
done 👍